packages/Teams/Microsoft.Bot.Components.Teams.csproj

```diff
@@ -12,10 +12,14 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <PackageId>Microsoft.Bot.Components.Teams</PackageId>
+    <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
```xml
<Version Condition=" '$(PreviewPackageVersion)' == '' ">$(LocalPackageVersion)</Version>
    <Version Condition=" '$(PreviewPackageVersion)' != '' ">$(PreviewPackageVersion)</Version>
    <PackageVersion Condition=" '$(PreviewPackageVersion)' == '' ">$(LocalPackageVersion)</PackageVersion>
    <PackageVersion Condition=" '$(PreviewPackageVersion)' != '' ">$(PreviewPackageVersion)</PackageVersion>
```
This can be removed from above (sorry i missed this. I think it is breaking the build?)

Reply:
Removed - it's passing now.